Helper/PaymentMethods.php

```diff
@@ -170,10 +170,10 @@ public function getAdyenPaymentCodes(): array
     public function isAdyenPayment(string $methodCode): bool
     {
         if(in_array($methodCode, $this->getAdyenPaymentCodes())) {
-            return false;
+            return true;
         }
 
-        return true;
+        return false;
```

Comment:
Would it be easier to `return in_array($methodCode, $this->getAdyenPaymentCodes());` ?

Reply:
Thank you, changed.